src/services/location.service.js

```diff
@@ -16,8 +16,8 @@ class LocationService {
  * @returns {object} data
  */
   static findLocationByProperty(property) {
-    return Location.findOne({
-      where: { ...property }
+    return Location.findAll({
```

Comment:
Why did you change to findAll, doesn't it affect other feature?

Reply:
Ok @victor-abz , i created two so that i don't override someone else's task